verify.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, configfile, auth):
             elif opts[line] == '-d':
                 self.desc = opts[line + 1]
             elif opts[line] == '-b':
-                self.bin = opts[line + 1]
+                self.bin = self.desc
```

Comment:
@ruebot, thanks!  I think you are right that it really is that simple.  One caveat, can we be sure that the config file will always have the -d before the -b?  I'm wondering if the order is fixed or does it depend on the order entered at the command line?